db/mysql/mysqlClient.go

```diff
@@ -0,0 +1,140 @@
+package mysql
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/go-sql-driver/mysql"
+	guuid "github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/spf13/viper"
+
+	"github.com/abdulmoeid7112/student-backend-portal/config"
+	"github.com/abdulmoeid7112/student-backend-portal/db"
+	"github.com/abdulmoeid7112/student-backend-portal/models"
+)
+
+//ErrorFooUserNotFoundError.
+var ErrorFooUserNotFoundError = errors.New("User not found")
```

Comment:
I don't like this approach to errors... You are duplicating errors for both mongo and mysql.. and what is `FOO` here?

Use this approach instead: https://github.com/ahab94/ApnaSchool/tree/dev/errors